src/Workspaces/Core/Portable/Workspace/Solution/StateChecksums.cs

```diff
@@ -168,11 +169,17 @@ public ProjectStateChecksums(params object[] children) : base(WellKnownSynchroni
                 result[AdditionalDocuments.Checksum] = AdditionalDocuments;
             }
 
+            if (searchingChecksumsLeft.Remove(AnalyzerConfigDocuments.Checksum))
+            {
+                result[AnalyzerConfigDocuments.Checksum] = AdditionalDocuments;
```

Comment:
AnalyuzerConfigDocumetns?

Reply:
hmmm no test failed?

Reply:
looks like test hole here. everyone uses ProjectChecksum.AnalyzerConfigDocuments directly so no one ever ask to find this from checksum.

Reply:
I will add test when I made solution creation part to be shared between test and solution creator.